packages/amplify_auth_cognito/lib/src/CognitoSession/AWSCredentials.dart

```diff
@@ -23,18 +23,18 @@ class AWSCredentials {
   String awsSecretKey;
   String sessionToken;
   AWSCredentials.init({@required LinkedHashMap<dynamic, dynamic> creds}) {
-    var realCreds;
-    if (Platform.isAndroid) {
-      if (creds.containsKey("value")) {
-        realCreds = creds["value"];
+    if (creds != null) {
+      var realCreds = {};
+      if (Platform.isAndroid) {
+        if (creds.containsKey("value")) {
+          realCreds = creds["value"];
+        }
       } else {
```

Comment:
I still see this check, is this necessary?

Reply:
This is a seperate attribute - but I've made the same fix.